common/courses/states.ts

```diff
@@ -76,6 +77,15 @@ export async function allowCourse(course: Course, screeningComment: string | nul
             await publishSubcourse(subcourse);
         }
     }
+
+    const subcourse = await prisma.subcourse.findFirst({ where: { courseId: course.id } });
```

Comment:
Can't we use the subcourses from line 74?

Reply:
I destructured the first element from the subcourses array.